packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -28,6 +28,8 @@ export const AlertConsumers = {
   STACK_ALERTS: 'stackAlerts',
   EXAMPLE: 'AlertingExample',
   MONITORING: 'monitoring',
+  ALERTS: 'alerts',
+  DISCOVER: 'discover',
```

Comment:
Now that the consumers are decoupled from the feature IDs the list should include all possible consumers so far. `alerts` and `discover` are valid ones. Ideally, we should not have a list of possible consumers. I hope in the subsequent PRs we will remove it.